src/custom-elements/folder-info-container/folderInfoContainer.js

```diff
@@ -0,0 +1,127 @@
+export class FolderInfoContainer extends HTMLElement {
+  // 컴포넌트 정보를 담고 있는 프로퍼티
+  #prop = null;
```

Comment:
```suggestion
  _prop = null;
```
#을 변수이름에서 사용하는건 js에선 좀 생소해보여요.

Reply:
@flynncodeit ES2022부터 클래스 안에서 private property를 지정하는 문법으로 쓰인다고 알게 되었습니다! 너무 최신 문법이라 아직 많이 쓰이지는 않는 것 같습니다..ㅜㅜ
[private property in js](https://dev.to/smitterhane/private-class-fields-in-javascript-es2022-3b8)